setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='COVID-19 Reproduction Number',
+    version='0.0.1',
+    url='https://github.com/ResearchLuxembourg/covid-19_reproductionNumber',
+    author='Research Luxembourg WP6',
+    author_email='lcsb-r3@uni.lu',
+    description='Estimator for COVID-19 R(t)',
+    install_requires=[
+        'pandas ',
+        'numpy ',
+        'datetime ',
+        'matplotlib ',
+        'scipy ',
+        'openpyxl'
+    ],
```

Comment:
if we have these, then we don't really need `requirements.txt` (`pip install -e .` will find these packages in setup.py).

Moreover - it'd be good to pin the versions here

Reply:
Okay, what would be easier to maintain?

Reply:
IMO setup.py to have the requirements listed only once